cmake/ot_libs_configuration_write.cmake

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+include(${NRFXLIB_DIR}/common.cmake)
```

Comment:
This is actually a bad pattern, but I know it's used in nrfxlib also.

But more important. then there is a risk associated here.
This code assumes that `nrfxlib` Zephyr module has been processed first, which may not always be the case.

But you probably noticed that when you made the sample, which is why you have this in the sample.
https://github.com/nrfconnect/sdk-nrf/blob/200decf54c3ef0e5925dde8cd98946134d5f3f68/samples/openthread/cli/CMakeLists.txt#L17